src/main/java/core/basesyntax/TextFiles.java

```diff
@@ -0,0 +1,168 @@
+                break;
+            case 4:
+                help();
+                break;
+            case 5:
+                help(args[1]);
+                break;
+            default:
+                exit();
+                break;
+        }
+    }
+
+    public static void create(String path, String fileName) throws IOException {
+        try {
+            if (Files.exists(Paths.get(path + File.separator + fileName))) {
+                System.out.println("File already exists, do you want to rewrite it?  Y / N");
+                Scanner sc = new Scanner(System.in);
+                if (sc.toString().matches("Y")) {
+                    Files.delete(Paths.get(path + File.separator + fileName));
+                } else if (sc.toString().matches("X")) {
+                    exit();
+                } else {
+                    System.out.println("Please enter the correct command");
+                }
+            }
+            Files.createFile(Paths.get(path + File.separator + fileName));
+        } catch (IOException e) {
+            if (!Files.exists(Paths.get(path))) {
+                throw new IOException("The src dir doesn't exist");
+            }
+        }
+    }
+
+    public static void read(String path, String fileName) throws IOException {
+        try {
+            List<String> lines = Files.readAllLines(Paths.get(path + File.separator + fileName));
+            for (String s: lines) {
+                System.out.println(s);
+            }
+
+        } catch (IOException e) {
+            if (!Files.exists(Paths.get(path))) {
+                throw new IOException("The src dir doesn't exist");
+            }
+            if (!Files.exists(Paths.get(path + File.separator + fileName))) {
+                throw new IOException("The file doesn't exist");
+            }
+        }
+    }
+
+    public static void info(String path, String fileName) throws IOException {
+        Path path1 = Paths.get(path + File.separator + fileName);
+        List<String> lines = Files.readAllLines(Paths.get(path + File.separator + fileName));
+        int charsCount = 0;
+        int linesCount = 0;
+        int wordsCount = 0;
+        double fileSize = Files.size(path1);
+        FileTime dataTimeLastModification = Files.getLastModifiedTime(path1);
+        for (String s: lines) {
+            linesCount++;
+            String[] words = s.split(" ");
+            int wordsCountTemp = 0;
+            int charsCountTemp = 0;
+            wordsCount += wordsCountTemp;
+            charsCount += charsCountTemp;
+            for (String w: words) {
+                wordsCountTemp++;
+                charsCount += w.toCharArray().length;
+            }
+        }
+        System.out.println("There are " + charsCount + " chars" + "%n"
+                + "lines " + linesCount + "%n" + "words" + wordsCount + "%n"
+                + "The last modified sate was: " + dataTimeLastModification
+                + "%n" + "File size " + fileSize);
```

Comment:
We can simplify this using nio Files.readAllLines(file).size() = lines.And you can put it right into your sout. Google method for other cases)